process/outportBlockConverter.go

```diff
@@ -535,7 +674,71 @@ func (o *outportBlockConverter) HandleShardOutportBlock(outportBlock *outport.Ou
 	return shardOutportBlock, nil
 }
 
-func (o *outportBlockConverter) copyTransactions(sourceTxs map[string]*outport.TxInfo, transactionPool *hyperOutportBlocks.TransactionPoolV2) error {
+func (o *outportBlockConverter) copyTransactions(sourceTxs map[string]*outport.TxInfo) (map[string]*hyperOutportBlocks.TxInfo, error) {
```

Comment:
for `copyTransactions` we return txs and for `copyTransactionsV2` we pass the map and modify it, can we have the same approach?